backend/src/api_parsers/dblp_parser.py

```diff
@@ -59,11 +59,13 @@ def _parse_hit_dict(self, hit: dict) -> None:
             split_name = author_name.split()
             first_name, last_name = split_name[0], " ".join(split_name[1:])
             auth_data = {
+                "author_external_id": author_id,
                 "first_name": first_name,
                 "last_name": last_name,
-                "api_id": author_id,
-                "publication": publication,
                 "affiliation": None,
+                "email": None,
+                "source": Source.DBLP,
+                "publication": publication,
```

Comment:
Each author can have only one publication?

Reply:
in our case yes, since we're extracting author data from publications returned by the API response. so an author can occur multiple times under different publications, however each publication is of different value to the user (abstract similarity, year, etc.) and so does the author associated with the publication. gathering multiple publications under one author would make it difficult and unnecessarily complicated to evaluate score of a result.